syntax = "proto3";

service TestService {
  rpc TestProto (Test) returns (Test) {}
}

message Test {
  optional bool testBool = 1;
}
